osi_object.proto

```diff
@@ -745,66 +748,133 @@ message MovingObject
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
             //
             TYPE_HEAVY_TRUCK = 7;
 
-            // Vehicle is a truck with semitrailer.
+            // Vehicle is a tractor unit capable of pulling a semi-trailer.
+            //
+            // Definition: The cab for an articulated vehicle.
+            //
+            // This has no equivalent in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
+            //
+            TYPE_SEMITRACTOR = 16;
+
+            // This vehicle is a trailer unit for an articulated vehicle.
+            //
+            // Definition: A trailer that can be pulled by a semi-tractor.
+            // There is no precise definition of this vs a typical trailer, although
+            // typically it is used for trailers with a non-trivial bounding box
+            // overlap with the tractor unit.
+            //
+            // This corresponds to "semitrailer" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "trailer" in OpenScenario 2.x.
             //
             TYPE_SEMITRAILER = 8;
 
             // Vehicle is a trailer (possibly attached to another vehicle).
             //
+            // Definition: A trailer that isn't pulled by a semi-tractor.
+            //
+            // This corresponds to "trailer" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "trailer" in OpenScenario 2.x.
+            //
             TYPE_TRAILER = 9;
 
             // Vehicle is a motorbike or moped.
             //
+            // This corresponds to "motorbike" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "VRU_vehicle" in OpenScenario 2.x.
+            //
             TYPE_MOTORBIKE = 10;
 
             // Vehicle is a bicycle (without motor and specific lights).
             //
+            // This corresponds to "bicycle" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "bus" in OpenScenario 2.x.
+            //
             TYPE_BICYCLE = 11;
 
             // Vehicle is a bus.
             //
+            // This corresponds to "bus" in both OpenScenario 1.x and 2.x.
+            //
+            // This is one of the OSI types which map from "bus" in OpenScenario 2.x.
+            //
             TYPE_BUS = 12;
 
             // Vehicle is a tram.
             //
+            // This corresponds to "tram" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "bus" in OpenScenario 2.x.
+            //
             TYPE_TRAM = 13;
 
             // Vehicle is a train.
             //
+            // This corresponds to "train" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "bus" in OpenScenario 2.x.
```

Comment:
I am not sure OpenSCENARIO 2.0 intends to describe a tram with bus. Did you talk with the Domain Model people there? While things are still under development there, they currently have an `intended_infrastructure` enum with the value `rail`while (you are right here) no value in `vehicle_category` for train. Unless clarified I think I would delete this line here.

Reply:
See comment on trams.